features/yield/FarmingCalculator.tsx

```diff
@@ -167,33 +181,58 @@ const FarmingCalculator = () => {
 
   // Update yield amount whenever inputs change.
   useEffect(() => {
-    const updatedYield = calculateYield(
+    const updatedUmaYield = calculateYield(
       Number(yUSDAdded) || 0,
       Number(USDCAdded) || 0,
       Number(yUSDPrice) || 0,
       Number(poolLiquidity) || 0,
       Number(umaPrice) || 0,
```

Comment:
You should add `Number(renPrice) || 0` here and change the default value for `renPrice` back to ""